tests/OSPSuite.Core.Tests/Core/TableFormulaWithXArgumentSpecs.cs

```diff
@@ -109,7 +109,67 @@ protected override void Context()
       [Observation]
       public void should_throw_an_exception_containing_the_alias()
       {
-         The.Action(() => CalcValue()).ShouldThrowAn<Exception>();
+         The.Action(() =>
+         {
+            var v = _parameter.Value;
+         }).ShouldThrowAn<Exception>();
+      }
+   }
+
+   public class When_a_parameter_is_usung_a_table_formula_with_x_argument_and_resolved_object_and_the_x_argument_changes : concern_for_TableFormulaWithXArgument
+   {
+      protected override void Context()
+      {
+         base.Context();
+         sut.ResolveObjectPathsFor(_parameter);
```

Comment:
Key to make the test fail and then pass. We need to ResolveObjectPath in order to trigger caching